roxie/ccd/ccdserver.cpp

```diff
@@ -5210,11 +5219,15 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
             else
             {
                 unsigned timeNow = msTick();
-                if (!anyActivity && !localAgent && (timeNow-lastActivity >= timeout))
+                if (acknowledgeAllRequests)
```

Comment:
Should this also be nested in an if (localAgent)?

Reply:
Probably wouldn't hurt... I will push a fix